config/initializers/acts_as_tenant.rb

```diff
@@ -0,0 +1 @@
+require "acts_as_tenant"
```

Comment:
@eclarizio Can this be moved to app/models/application_record.rb and remove this file.